README.md

```diff
@@ -164,63 +91,34 @@ workspace and select **View details**. Save this ID for later.
 
 ![](doc/source/images/dialog.PNG)
 
-## 4. Configure Watson Retrieve and Rank
-
-Launch the **Watson retrieve-and-rank** tool. Create a **new data cluster**.
-
-![](doc/source/images/rnr-cluster.PNG)
-
-Seed the content by firstly creating **New Collection**, and add the file documents and questions present under [`data/Retrieve&Rank/`](data/Retrieve&Rank/)
-
-![](doc/source/images/new-collection.PNG)
-
-> Note: Ensure that you have also created a [**Watson Document Conversion**](https://console.ng.bluemix.net/catalog/services/document-conversion) service as well. Since, Watson RnR uses document conversion at the backend.
-
-## 5. Run the application
+### 4. Load the Discovery documents
 
-### If you used the Deploy to Bluemix button...
+Launch the **Watson Discovery** tool. Create a new environment if prompted. Create a new Collection.
 
-If you used ``Deploy to Bluemix``, most of the setup is automatic to start using the App.
+###### Save the COLLECTION_ID for later.
 
-### If you want to run it locally...
-
-Otherwise as explained above:
-1. Clone the repo
-2. Instantiate/ Configure the above watson services
-2. Install NodeJS runtime or npm.
-3. Start the app and it can be used at localhost:3000
+Add the file documents and questions present under [`data/Retrieve&Rank/`](data/Retrieve&Rank/)
```

Comment:
Agree. Ampersand in a dir name really bugs me... but it works. Probably will be moving file to "discovery".